docs/core/docker/build-container.md

```diff
@@ -283,7 +283,18 @@ Save the *Dockerfile* file. The directory structure of the working folder should
 
 :::zone-end
 
-From your terminal, run the following command:
+The `ENTRYPOINT` instruction sets `dotnet` as the host for the `DotNet.Docker.dll`. However, it's possible to instead define the `ENTRYPOINT` as the app executable itself, relying on the OS as the app host:
+
+```dockerfile
+ENTRYPOINT ["./DotNet.Docker"]
+```
+
+This causes the app to be executed directly, without `dotnet`, and instead relies on the app host and the underlying OS. There should be no app behavior difference when launching from via app host versus the `dotnet` host for ordinary server apps. Launching via an app host gives you better integration with the underlying OS. For example:
+
+- You'll see the app host name in your process list and not `dotnet`, which could be confusing if there is more than one.
+- You can customize the app host with OS specific features. For more information, see [this discussion about configuring default stack size on Windows](https://github.com/dotnet/runtime/issues/96347#issuecomment-1981470713).
```

Comment:
```suggestion
- You can customize the app host with OS specific features. For example, see [this discussion about configuring default stack size on Windows](https://github.com/dotnet/runtime/issues/96347#issuecomment-1981470713).
```
The discussion about the stack size is just one example of the customization. It is not a comprehensive discussion of all customizations possible.